css-canvas-medical/styles.css

```diff
@@ -0,0 +1,227 @@
+p {
+  margin: 0;
+  font-family:
+    Avenir,
+    sans-serif;
+}
+
+.container {
+  max-width: 80%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo > img {
+  width: 170px;
+}
+
+ul {
+  padding: 0;
+}
+
+ul li {
+  display: inline-block;
+  font-size: 15px;
+  padding: 10px;
+  font-family: Avenir, sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: #b7b7b7;
+}
+
+li :hover {
+  color: #294464;
+}
+
+#navbtn {
+  background-color: #294464;
+  color: white;
+  font-size: 15px;
+  font-family:
+    Avenir,
+    sans-serif;
+  border-style: none;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+hr {
+  position: relative;
+  bottom: 15px;
+  height: 1px;
+  border: 0;
+  background-color: #eaecef;
+}
+
+.backgroundimg {
+  background-image: linear-gradient(rgb(0 0 0 / 20%), rgb(0 0 0 / 30%)), url("images/hero.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  margin-left: auto;
+}
+
+.col-one-half {
+  width: 60%;
+  padding: 80px;
+}
+
+.col-one-half > h1 {
+  color: white;
+  font-weight: normal;
+}
+
+.col-one-half > p {
+  color: white;
+  line-height: 1.6;
+  font-family: Avenir, sans-serif;
+  font-size: 15px;
+}
+
+.col-one-half > button {
+  border-radius: 5px;
+  width: 200px;
+}
+
+#btn {
+  background-color: #294464;
+  color: white;
+  font-size: larger;
+  font-family:
+    Avenir,
+    sans-serif;
+  border-style: none;
+  padding: 15px;
+  margin-top: 30px;
+}
+
+.locations {
+  background-color: #fafafa;
+  margin-top: 40px;
+  padding-top: 40px;
+  padding-bottom: 60px;
+}
+
+.loc-one-half {
+  width: 50%;
+  font-family:
+    Avenir,
+    sans-serif;
+}
+
+.align-content-center {
+  text-align: center;
+  margin: auto;
+  width: 40%;
+  padding-bottom: 60px;
+}
+
+.align-content-center > h2 {
+  margin-bottom: 15px;
+  color: #858585;
+  font-size: 25px;
+  font-weight: normal;
+  font-family: "Tiempos Headline", sans-serif;
+}
+
+.align-content-center > p {
+  line-height: 1.6;
+  color: #909090;
+}
+
+.loc-one-half > h3 {
+  text-align: center;
+  font-weight: normal;
+  font-size: 15px;
+  padding: 10px;
+  color: #909090;
+}
+
+.container-width {
+  max-width: 80%;
+  margin: auto;
+}
```

Comment:
This should just be your plain `container` class

Reply:
I see that this _already is_ your plain `container` class, on line 8. Be careful not to unnecessarily duplicate styling 👀